src/utils/EthDataProvider/EthDataProvider.js

```diff
@@ -172,21 +177,17 @@ const reverseIQtoEOSTx = async (amount, wallet, eosAccount) => {
   const amountParsed = ethers.utils.parseEther(amount).toString();
   if (wallet.status === "connected") {
     const provider = new ethers.providers.Web3Provider(wallet.ethereum);
-    const erc20 = new ethers.Contract(
-      iqAddress,
-      erc20Abi,
-      provider.getSigner()
-    );
+
+    const erc20 = getERC20IQContract(wallet);
```

Comment:
this was wrong. Please make sure you test what you refactor. Bridge ETH -> EOS was broken